scene/gui/scroll_container.h

```diff
@@ -72,13 +72,18 @@ class ScrollContainer : public Container {
 
 	struct ThemeCache {
 		Ref<StyleBox> panel_style;
+		Ref<StyleBox> focus_style;
 	} theme_cache;
 
 	void _cancel_drag();
 
 	bool _is_h_scroll_visible() const;
 	bool _is_v_scroll_visible() const;
 
+	bool _draw_focus_border = false;
+	bool _focus_border_is_drawn = false;
```

Comment:
```suggestion
	bool draw_focus_border = false;
	bool focus_border_is_drawn = false;
```
Only private methods are usually prefixed.